### ISSUE TITLE
io.quarkus:quarkus-universe-bom:1.1.0.CR1 Not Released on Maven Central Repo

### ISSUE THREAD

**Issue by mingjliu9** [quarkusio/quarkus#6171]
**Describe the bug**
version 1.1.0.CR1 couldn't be found on maven central repository

**Expected behavior**
Should show up in the page, just as 1.0.0.CR1 as shown in screenshots.

**Actual behavior**
version 1.1.0.CR1 couldn't be found with following link

**To Reproduce**
1. Created project created on https://code.quarkus.io/
2. update quarkus platform version in pom to 1.1.0.CR1
```
<quarkus.platform.version>1.1.0.CR1</quarkus.platform.version>
```
3. build will be failed 
4. Follow the link https://search.maven.org/artifact/io.quarkus/quarkus-universe-bom and found no such version published.

**Screenshots**

![image](https://user-images.githubusercontent.com/39477523/70846719-028e5a00-1e97-11ea-9853-737f39480c93.png)

**Comment by gwenneg** [comment]
Hi @mingjliu9 and thanks for testing Quarkus 1.1.0.CR1!

If I'm not mistaken, the universe bom is not released for a CR and will be release with `1.1.0.Final`.

@gsmet will know better than me about this, here's something he wrote:

> We have only released the core Quarkus, not the platform, so:
>- on an existing project, you need to use the quarkus-bom instead of the quarkus-universe-bom. On recent projects, it's usually defined in the `quarkus.platform.artifact-id` property of your pom.xml. And, of course, update the version.
>- to create a new project, use:
mvn io.quarkus:quarkus-maven-plugin:1.1.0.CR1:create -DplatformVersion=1.1.0.CR1 -DplatformArtifactId=quarkus-bom

**Comment by gsmet** [comment]
Yes, for candidate releases, from now one, we will only release the core Quarkus and not the platform and the universe bom.

The universe bom contains Camel for instance and we need to give them the time to test the CR too.

See https://groups.google.com/d/msg/quarkus-dev/rap-O3OT4k4/n8XVaT1LAgAJ for info about how to use it.
